package.json

```diff
@@ -8,37 +8,42 @@
     "type": "git",
     "url": "github:philips-software/odin"
   },
+  "type": "module",
   "main": "lib/index.js",
   "files": [
     "lib"
   ],
   "scripts": {
-    "build": "tsc --build",
+    "build": "tsc --build tsconfig-refs.json",
     "clean": "rimraf --glob lib **/.eslintcache **/*.log **/*.tsbuildinfo **/coverage",
     "lint": "eslint . --ext .js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --cache --max-warnings 0",
     "release": "semantic-release",
     "test": "vitest run --coverage",
     "test:dev": "vitest dev --coverage",
     "test:ui": "vitest dev --coverage --ui"
   },
+  "dependencies": {
+    "debug": "^4.3.4"
```

Comment:
pin dependency;

Reply:
open source packages should not pin deps, it hinders the users ability to get security fixes